doc/introduction/interfaces/jax.rst

```diff
@@ -196,3 +196,75 @@ Example:
 
     If you don't pass a PRNGKey when sampling with a ``jax.jit``, every call to the sample function
     will return the same result.
+
+.. _jax_optimize:
+
+Optimization using JAXopt and Optax
+-----------------------------------
+
+To optimize your hybrid classical-quantum model using the JAX interface, you
+**must** make use of a package meant for optimizing JAX code (such as `JAXopt
+<https://jaxopt.github.io/stable/>`_ or `Optax

```

Comment:
I prefer the github links directly, but only personal preference. e.g. https://github.com/google/jaxopt

Reply:
Interesting! We seem to be linking to the website for other interfaces, so I might leave it as is, but that's nice to know. :+1: